base_user_groups_dafa/security/security.xml

```diff
@@ -41,7 +41,10 @@
         <field name="comment">DAFA Superadmin</field> # Allows users to manage/edit the information related to the organization and all its departments.
```

Comment:
Jag gissar på att felet finns här någonstans. Fast det kanske finns i error i någon annan xml i denna PR

Reply:
This error resolved with, when https://github.com/vertelab/odoo-base/pull/208 is with the branch